crates/symbolicator-proguard/src/symbolication.rs

```diff
@@ -74,6 +117,10 @@ impl ProguardService {
                     .flat_map(|frame| {
                         Self::map_frame(&mappers, frame, release_package.as_deref()).into_iter()
                     })
+                    .map(|mut frame| {
+                        Self::apply_source_context(&source_bundle_sessions, &mut frame);
```

Comment:
source context application should be optional, similar to how it is for native / JS, because profiling is never using source context.

Reply:
Thank you, completely forgot about that.